### DISCUSSION TITLE
Ajustements BD

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#52] [Show and tell]
Si vous avez des erreurs:
ALTER TABLE [nom_de_la_table]
MODIFY creation_date TIMESTAMP NOT NULL 
DEFAULT CURRENT_TIMESTAMP;

**Comment by NabilDjerroud** [comment]
@CollRen , s'il te plait une question par rapport à l'insertion des données dans la bd, je voulais savoir si c'est correct comment j'ai inséré les données dans la tabe Carburant ? voir la capture d'écran 
![image](https://github.com/582-41E-MA/projet-web-2-CollRen/assets/144846468/ab37b40f-180c-4922-9940-06d4b215d2af)

**Reply by CollRen** [reply]
Oui c'est bon

**Reply by NabilDjerroud** [reply]
cool merci
